.pipelines/containerSourceData/scripts/BuildContainerCommonSteps.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+# Add for debuggin purposes
```

Comment:
Is this comment needed? Also, since we are echoing some variables here, can we echo all the ones being using this script?